frontend/src/components/account/inventory/Inventory.js

```diff
@@ -12,16 +12,156 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */
 import React, {Component} from 'react';
+import InventoryItems from "./InventoryItems";
+import { loading } from "../../utils/Utilities";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import { Link } from "react-router-dom";
+import "./Inventory.css";
+import ComponentWithHeader from "../../header/ComponentWithHeader";
 
-class Inventory extends Component {
+class Inventory extends ComponentWithHeader {
+  constructor(properties) {
+    super(properties);
+    this.state = {
+      inventory: [],
+      edit: false,
+    };
+
+    this.addItem = this.addItem.bind(this);
+    this.deleteItem = this.deleteItem.bind(this);
+  }
+  
+  setEdit() {
+    this.getInventory();
+    const previousEdit = this.state.edit;
+    this.setState({edit: !previousEdit});
+  }
+
+  editLabel(){
+      if (this.state.edit){
+        return "Cancel";
+      } else {
+        return "Edit"
+      }
```

Comment:
Forgot to write that before, but you can use a ternary operator here and in editCssLabel

`return this.state.edit ? "Cancel" : "Edit";`